### PR TITLE
Copy author's license to oackage folder

### PR DESCRIPTION
**Pull request by uilianries**
related issue: https://github.com/bincrafters/community/issues/615

- Fix license copy in package folder
- Remove libcxx from settings
- Apply Conan convetions

### CODE REVIEW

**Comment by uilianries** [comment]
The Bintray page is up to date, but we will need to remove all outdated packages after to merge it. fPIC was not present before, so we will have multiple flavors of same package.

**Review by danimtb — APPROVED** [review]

File: conanfile.py [hunk 1]
```diff
@@ -20,25 +20,39 @@ class NanomsgConan(ConanFile):
...
+    _source_subfolder = "source_subfolder"
 
-    default_options = {'shared': False, 'enable_doc': False, 'enable_getaddrinfo_a': True, 'enable_tests': False, 'enable_tools': True, 'enable_nanocat': True}
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
 
-    _source_subfolder = "source_subfolder"
+    def configure(self):
+        del self.settings.libcxx
```
Comment (SSE4) [inline conanfile.py]: it probably must be `self.settings.compiler.libcxx`?

Comment (uilianries) [reply conanfile.py]: Fixed!

**Pull request merged**